includes/shortcodes/class.llms.shortcodes.php

```diff
@@ -34,9 +35,9 @@ public static function init() {
 			/**
 			 * Filters the shortcodes to initialize
 			 *
-			 * @since unknown
+			 * @since Unknown.
```

Comment:
```suggestion
			 * @since Unknown
```

Unknown as the description should have a period, Unknown as the version should not have a period

Eg:

```
@since Unknown
@since 1.0.0 Unknown.
```